### PR TITLE
fix(DB/Visual): The Sun Gate

### PR DESCRIPTION
**Pull request by wizzymore**
## Changes Proposed:
-  Add correct visuals to The Sun Gate

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4094

## Tests Performed:
- Ubuntu 20.04

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by Kitzunu** [comment]
image
Hmm :PP jk ofc

**Comment by wizzymore** [comment]
Yeah, it's something annoying about the gameobject not being able to activate after the spell cast, and because of this i cannot remove the DND creature.. I will add the file anyway, mark this as WIP.

**Comment by wizzymore** [comment]
Ready for review now :D

File: data/sql/updates/pending_db_world/rev_1609274862007339479.sql [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+(-63583, 0, 0, 0, 11, 0, 100, 0, 0, 0, 0, 0, 0, 11, 31411, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, '[DND]Sunhawk Portal Controller - On Respawn - Cast \'Stabilize Sun Gate I\''),
+(-63584, 0, 0, 0, 11, 0, 100, 0, 0, 0, 0, 0, 0, 11, 31412, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, '[DND]Sunhawk Portal Controller - On Respawn - Cast \'Stabilize Sun Gate II\''),
+(-63585, 0, 0, 0, 11, 0, 100, 0, 0, 0, 0, 0, 0, 11, 31413, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, '[DND]Sunhawk Portal Controller - On Respawn - Cast \'Stabilize Sun Gate III\''),
+(-63582, 0, 0, 0, 11, 0, 100, 0, 0, 0, 0, 0, 0, 11, 31414, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, '[DND]Sunhawk Portal Controller - On Respawn - Cast \'Stabilize Sun Gate IV\'');
+
+UPDATE `gameobject_template` SET `AIName` = 'SmartGameObjectAI' WHERE `entry` IN (184850, 182026);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1609274862007339479.sql:12]: Why update 184850? You dont add a script for it

File: data/sql/updates/pending_db_world/rev_1609274862007339479.sql [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+UPDATE `creature` SET `position_x`=-2147.350098, `position_y`=-10741.099609, `position_z`=73.903397 WHERE `guid`=63583;
+UPDATE `creature` SET `position_x`=-2128.939941, `position_y`=-10726.000000, `position_z`=66.335800 WHERE `guid`=63584;
+UPDATE `creature` SET `position_x`=-2107.070068, `position_y`=-10703.000000, `position_z`=65.189400 WHERE `guid`=63585;
+UPDATE `creature` SET `position_x`=-2098.620117, `position_y`=-10688.500000, `position_z`=65.218102 WHERE `guid`=63582;
+
+UPDATE `creature` SET `spawntimesecs` = 33 WHERE `id` = 17886;
+UPDATE `gameobject` SET `spawntimesecs` = 30, `animprogress` = 100 WHERE `id` = 184850;
+UPDATE `gameobject_template` SET `Data3` = 3000 WHERE (`entry` = 184850);
+
+DELETE FROM `creature` WHERE `guid` IN (96886);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1609274862007339479.sql:43]: and ID

File: data/sql/updates/pending_db_world/rev_1609274862007339479.sql (hunk 1 above)
Comment (wizzymore) [reply data/sql/updates/pending_db_world/rev_1609274862007339479.sql:12]: The scrips are added on guid

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1609274862007339479.sql:12]: Ah

File: data/sql/updates/pending_db_world/rev_1609274862007339479.sql (hunk 2 above)
Comment (wizzymore) [reply data/sql/updates/pending_db_world/rev_1609274862007339479.sql:43]: I don't need to delete all of them, i am adding a new spawn alongside the 3 already existent

**Comment by pklloveyou** [comment]
Tested

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
@pklloveyou welcome back dude!

good job @wizzymore

**Pull request merged**